site/layouts/partials/sidebar.html

```diff
@@ -1,6 +1,6 @@
 <div id="sidebar">
   <a href="{{.Site.BaseURL}}" id="sidebar-logo"><img src="{{ "logo/logo.png" | relURL }}" width="100%" /></a>
-  <div style="background-color: rgba(0, 0, 0, 0.1); width: 100%; height: 1px"></div>
+  <div style="background-color: rgba(0, 0, 0, 0.1); width: 100%; height: 1px; min-height: 1px;"></div>
```

Comment:
without this it's compressed to zero on resize, unrelated to the original PR, but I noticed it while testing the scroll behavior